orisa/orisa.py

```diff
@@ -1846,7 +1846,7 @@ async def _message_create(self, ctx, msg):
     @event("guild_leave")
     async def _guild_leave(self, ctx, guild):
         logger.info(
-            "I was removed from guild %s, I'm now in %d guilds",
```

Comment:
These and other log messages are written this way because `guild` is an internal object, it will look like `removed from guild <Guild(id=12345678, name="foo">`. The alternative, `removed from the <Guild(...)> guild` feels weird to me. Also, the original has the advantage that I can grep through the log file searching for "removed from guild" instead of having to do "removed from .* guild"

Reply:
Fair point, I thought it was the actual name of the guild.